controllers/articles.controller.js

```diff
@@ -10,3 +10,15 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { incVotes } = req.body;
+  return incVotesById(incVotes, article_id)
+    .then((updatedArticle) => {
+      res.status(201).send({ article: updatedArticle });
```

Comment:
It should return a status 200 (not 201). This is because it's just returning you the final article. 201 means it creates a completely new 'thing', which in this case it isn't... it's just updating an existing 'thing'.